projects/storefrontlib/cms-components/misc/theme-switcher/config/default-theme-switcher-config.ts

```diff
@@ -0,0 +1,23 @@
+/*
+ * SPDX-FileCopyrightText: 2024 SAP Spartacus team <spartacus-team@sap.com>
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+import { ThemeSwitcherConfig } from './theme-switcher-config';
+
+export const defaultThemeSwitcherConfig: ThemeSwitcherConfig = {
+  themeSwitcher: {
+    themes: [
+      { i18nNameKey: 'themeSwitcher.themes.default', className: 'santorini' },
```

Comment:
Why did we hardcoded in our default config the default theme `santorini`?

Will it be a breaking change for existing customers updating Spartacus version, that don't use the `santorini` theme, but their own-named theme?